inspire_disambiguation/api.py

```diff
@@ -78,32 +81,42 @@ def train_and_save_distance_model(
             will be saved.
         sampled_pairs_size (int): Number of pairs to be generated for the training.
             Note:
-                Must be multiple of 12.
+                Must be multiple of 4.
     """
     LOGGER.info("Pulling training data from ES")
     start_time = datetime.now()
     curated_signatures = get_signatures(only_curated=True)
-    input_clusters = get_input_clusters(curated_signatures)
+    train_signatures_dict, test_signatures_dict = \
+        train_validation_split(curated_signatures, train_to_validation_split_fraction)
```

Comment:
can you avoid adding `\` at the end of lines? the right way to do this is
```suggestion
    train_signatures_dict, test_signatures_dict = train_validation_split(
        curated_signatures, train_to_validation_split_fraction
    )
```
you can also run `black` to auto-format.